### PR TITLE
update netlayer

### PR DESCRIPTION
**Pull request by preland**
This PR addresses #914. It is dependent on https://github.com/haveno-dex/netlayer/pull/3 being merged, which itself is dependent on https://github.com/haveno-dex/tor-binary/pull/4 being merged.

The current commit will not work correctly until these two are merged; after they are merged, the verification data can also be written and then committed.

### CODE REVIEW

**Comment by woodser** [comment]
The dependencies are committed.

**Comment by preland** [comment]
netlayer's commit hash was incorrect; it is now fixed in another PR

**Comment by woodser** [comment]
Merged.

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -281,36 +281,76 @@
             <sha256 value="e8f1e75d1355e2223d682224c45563b74da67f8638484960e04d7b2313919264" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.github.haveno-dex.netlayer" name="tor" version="72a4fae5cf">
```
Comment (woodser) [inline gradle/verification-metadata.xml:284]: Mind removing the old dependencies as well?

Comment (preland) [reply gradle/verification-metadata.xml:279]: Dependencies are removed

**Pull request merged**